Intersect (Core)/Config/Options.cs

```diff
@@ -146,6 +146,10 @@ public partial class Options
 
         public static bool ZDimensionVisible => Instance.MapOpts.ZDimensionVisible;
 
+        public static int MapItemHeight => Instance?.MapOpts?.MapItemHeight ?? 32;
+
+        public static int MapItemWidth => Instance?.MapOpts?.MapItemWidth ?? 32;
+
```

Comment:
Stop adding more statics

Reply:
offtopic and sort of related question, i had my doubts when doing this so, asking for later (a later chore PR perhaps): do we want to get rid of all these statics at Config/Options.cs eventually?